### PR TITLE
yapesdl: init at 0.70.2

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/emulators/yapesdl/default.nix [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  installPhase = ''
+    runHook preInstall
+    install --directory $out/bin $out/share/doc/$pname
+    install yapesdl $out/bin/
+    install README.SDL $out/share/doc/$pname/
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "http://yape.plus4.net/";
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/yapesdl/default.nix:35]:
```suggestion
    homepage = "https://yape.plus4.net/";
```

File: pkgs/misc/emulators/yapesdl/default.nix [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ];
+  buildInputs = [
+    SDL2
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    install --directory $out/bin $out/share/doc/$pname
+    install yapesdl $out/bin/
+    install README.SDL $out/share/doc/$pname/
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/yapesdl/default.nix:30]:
```suggestion
    install yapesdl -d $out/bin/
    install README.SDL -d $out/share/doc/$pname/
```

File: pkgs/misc/emulators/yapesdl/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/misc/emulators/yapesdl/default.nix:35]: The HTTP**S** site does not exist.

File: pkgs/misc/emulators/yapesdl/default.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/misc/emulators/yapesdl/default.nix:30]: Why?

Comment (SuperSandro2000) [reply pkgs/misc/emulators/yapesdl/default.nix:30]: Because it is shorter and install has a feature to create directories. Why not use it?

**Pull request merged**